dags/main_summary.py

```diff
@@ -734,13 +692,11 @@
 main_events.set_upstream(main_summary_export)
 main_events_bigquery_load.set_upstream(main_events)
 
-sql_main_summary_experiments.set_upstream(main_summary)
-sql_main_summary_experiments.set_upstream(main_summary_experiments_get_experiment_list)
-main_summary_experiments_export.set_upstream(sql_main_summary_experiments)
-main_summary_experiments.set_upstream(main_summary_export)
-main_summary_experiments_bigquery_load.set_upstream(main_summary_experiments)
+main_summary_experiments.set_upstream(main_summary)
+main_summary_experiments.set_upstream(main_summary_experiments_get_experiment_list)
+main_summary_experiments_export.set_upstream(main_summary_experiments)
 
-experiments_aggregates_import.set_upstream(main_summary_experiments)
+experiments_aggregates_import.set_upstream(main_summary_experiments_export)
```

Comment:
we can kill `experiments_aggregates_import`! It was used for the grafana experiment enrollment dashboards, which we switched to BQ this week :)

Reply:
done